ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -78,18 +64,19 @@ def create_bundle_build_file(build_out_file, lock_file, repo_name, excludes, wor
   bundle.specs.each { |spec|
     deps = spec.dependencies.map(&:name)
     deps += [":bundler_setup"]
-    template_out += GEM_TEMPLATE.gsub("{exclude}", excludes)
+
+    exclude_array = excludes[spec.name] || []
+    # We want to exclude files and folder with spaces in them
+    exclude_array += ["**/* *.*", "**/* */*"]
+
+    template_out += GEM_TEMPLATE.gsub("{exclude}", exclude_array.to_s)
```

Comment:
It might be been simpler to use `String#%`.

Reply:
I have cleaned up the excludes rules here https://github.com/bazelruby/rules_ruby/pull/38/files